docs/SETTINGS.rst

```diff
@@ -147,7 +147,17 @@ If you want that Patroni works with Etcd cluster via protocol version 3, you nee
 
 ZooKeeper
 ----------
--  **hosts**: list of ZooKeeper cluster members in format: ['host1:port1', 'host2:port2', 'etc...'].
+-  **hosts**: List of ZooKeeper cluster members in format: ['host1:port1', 'host2:port2', 'etc...'].
+-  **use_ssl**: (optional) Whether SSL is used or not. Defaults to `false`. If set to `false`, all SSL specific parameters are ignored.
```

Comment:
Similarly to `ENVIRONMENT.rst`, backslashes should be doubled, and a 'ca' -> 'CA' change would be nice

Reply:
Thank you. Docs updated